tests/src/test/scala/invokerShoot/ShootInvokerTests.scala

```diff
@@ -188,13 +189,13 @@ class ShootInvokerTests extends TestHelpers with WskTestHelpers with JsHelpers w
         JsObject("key" -> JsString("copiedAnnot2"), "value" -> JsBoolean(false)),
         JsObject("key" -> JsString("copiedAnnot1"), "value" -> JsString("copiedAnnotValue1")),
         JsObject("key" -> JsString("origAnnot2"), "value" -> JsBoolean(true)),
-        JsObject("key" -> JsString("exec"), "value" -> JsString("nodejs:6")),
+//        JsObject("key" -> JsString("exec"), "value" -> JsString(runtime)),
```

Comment:
Please drop this line.

Reply:
Sure, forgot that line… will be removed.